Biz/Action/UserLockout.pm

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023 bivio Software, Inc.  All rights reserved.
+package Bivio::Biz::Action::UserLockout;
+use strict;
+use Bivio::Base 'Biz.Action';
+
+my($_E) = b_use('Model.Email');
+
+sub execute_load_owner_email {
+    my(undef, $req) = @_;
+    $req->put(locked_owner_email => $_E->new($req)->unauth_load_or_die({
```

Comment:
I think values like this should never be put straight on the request. Rather, they should be put on `$self` and then referenced from there. This makes global name space management easier. See Action.Acknowledgement->_extract for an example (there are other suboptimal examples in there).